ethereum/executionlayer/build.gradle

```diff
@@ -16,5 +16,7 @@ dependencies {
   runtimeOnly 'io.jsonwebtoken:jjwt-impl'
   runtimeOnly 'io.jsonwebtoken:jjwt-jackson'
 
+  testImplementation 'com.squareup.okhttp3:mockwebserver'
```

Comment:
Given mockwebserver starts an actual server and network port, it would probably be better to make these integration tests.

Reply:
I got inspired by `MetricsPublisherTest`, which should be migrated to IT then

Reply:
Yeah definitely.  I've been meaning to find which test keeps opening a socket - MacOS pops up a dialog asking permission for it every time I run a build. :)